packages/fxa-content-server/app/scripts/lib/metrics.js

```diff
@@ -115,6 +118,19 @@ function marshallProperty(property) {
   }
 }
 
+function marshallUtmProperty(property) {
+  if (
+    property &&
+    property !== NOT_REPORTED_VALUE &&
+    Validate.isUtmValid(property)
+  ) {
+    if (Validate.isUtmValid(property)) {
```

Comment:
This syntax seems a little funny to me. Why are you checking `isUtmValid` twice?

Reply:
Whoops, removing